ml-agents/mlagents/trainers/tests/torch/test_policy.py

```diff
@@ -0,0 +1,151 @@
+import pytest
+from typing import Tuple
+
+import torch
+from mlagents.trainers.policy.torch_policy import TorchPolicy
+from mlagents.trainers.tests import mock_brain as mb
+from mlagents.trainers.settings import TrainerSettings, NetworkSettings
+from mlagents.trainers.torch.utils import ModelUtils
+
+VECTOR_ACTION_SPACE = 2
+VECTOR_OBS_SPACE = 8
+DISCRETE_ACTION_SPACE = [3, 3, 3, 2]
+BUFFER_INIT_SAMPLES = 32
+NUM_AGENTS = 12
+EPSILON = 1e-7
+
+
+def create_policy_mock(
+    dummy_config: TrainerSettings,
+    use_rnn: bool = False,
+    use_discrete: bool = True,
+    use_visual: bool = False,
+    seed: int = 0,
+) -> Tuple[TorchPolicy, TrainerSettings]:
```

Comment:
Why is it `Tuple[TorchPolicy, TrainerSettings]` instead of `TorchPolicy`?

Reply:
bug 😛